packages/spindle-ui/src/Pagination/hooks/useShowItem.ts

```diff
@@ -0,0 +1,47 @@
+import { useMemo } from 'react';
+
+type Payload = {
+  current: number;
+  total: number;
+  pageItem: number;
+};
+
+export function useShowItem({ current, total, pageItem = 5 }: Payload) {
+  const item: number[] = [];
+
+  const showItem = useMemo(() => {
```

Comment:
```suggestion
  const displayItem = useMemo(() => {
```
IMO. 細かいのですが、boolean or functionの印象を受けるため（ほかの同じプレフィックスの定数がbooleanでもあり）、命名を変更しても良さそうと思いました🙏